### ISSUE TITLE
Manifest release fails with Error: release-please failed: Not Found

### ISSUE THREAD

**Issue by wearethefoos** [google-github-actions/release-please-action#445]
### TL;DR

Two releases, one Elixir release from the root path (`.`) and another Node release from the `assets/` path fails with a not found error on the assets path.

One release, just the Elixir in the root path fails with `Error: release-please failed: Not Found`

### Expected behavior

I would expect the release action to not error and create releases for my root project and the assets project.

### Observed behavior

Instead, it fails with `Error: release-please failed: Not Found`

I tried with different paths: `assets` and `assets/` for the node release and with `.` for the Elixir release in the root folder. Nothing seems to work with the `manifest` command. I am pushing directly to the `main` branch to test this out, which is also the default branch for this repo.

### Action YAML

```yaml
name: Create Main Release

# Trigger the workflow on release activity
on:
  push:
    branches:
      - main

jobs:
  build:
    name: Create Release
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2

      - uses: GoogleCloudPlatform/release-please-action@v3
        id: release
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          command: manifest

      - name: Mark Deployment
        env:
          APPSIGNAL_PUSH_API_KEY: ${{ secrets.APPSIGNAL_PUSH_API_KEY }}
          REVISION: ${{ steps.release.outputs['packages/xxxxxx--tag_name'] }}
        run: |
          curl --silent --show-error --fail \
          -X POST -d "{\"revision\":\"${REVISION}\",\"repository\":\"${GITHUB_REF}\",\"user\":\"${GITHUB_ACTOR}\"}" "https://push.appsignal.com/1/markers?api_key=${APPSIGNAL_PUSH_API_KEY}&name=xxxxxx&environment=prod"
        if: ${{ steps.release.outputs.releases_created }}
```

### Log output

```text
2022-02-23T10:20:17.3329748Z ##[group]Run GoogleCloudPlatform/release-please-action@v3
2022-02-23T10:20:17.3330054Z with:
2022-02-23T10:20:17.3330465Z   token: ***
2022-02-23T10:20:17.3330667Z   command: manifest
2022-02-23T10:20:17.3330867Z   default-branch: main
2022-02-23T10:20:17.3331043Z   fork: false
2022-02-23T10:20:17.3331220Z   clean: true
2022-02-23T10:20:17.3331425Z   bump-minor-pre-major: false
2022-02-23T10:20:17.3331824Z   bump-patch-for-minor-pre-major: false
2022-02-23T10:20:17.3332067Z   monorepo-tags: false
2022-02-23T10:20:17.3332242Z   draft: false
2022-02-23T10:20:17.3332442Z   draft-pull-request: false
2022-02-23T10:20:17.3332644Z ##[endgroup]
2022-02-23T10:20:17.8664421Z ✔ Building pull requests
2022-02-23T10:20:17.8667063Z ✔ Building strategies by path
2022-02-23T10:20:18.3924044Z ##[error]release-please failed: Not Found


2022-02-23T10:20:03.9574638Z Requested labels: ubuntu-latest
2022-02-23T10:20:03.9574689Z Job defined at: Xxxxxx/xxxxxx/.github/workflows/release.yml@refs/heads/main
2022-02-23T10:20:03.9574713Z Waiting for a runner to pick up this job...
2022-02-23T10:20:04.9958723Z Job is waiting for a hosted runner to come online.
2022-02-23T10:20:08.5556640Z Job is about to start running on the hosted runner: Hosted Agent (hosted)
2022-02-23T10:20:13.4799989Z Current runner version: '2.287.1'
2022-02-23T10:20:13.4829049Z ##[group]Operating System
2022-02-23T10:20:13.4829750Z Ubuntu
2022-02-23T10:20:13.4830047Z 20.04.3
2022-02-23T10:20:13.4830720Z LTS
2022-02-23T10:20:13.4831046Z ##[endgroup]
2022-02-23T10:20:13.4831381Z ##[group]Virtual Environment
2022-02-23T10:20:13.4831795Z Environment: ubuntu-20.04
2022-02-23T10:20:13.4832134Z Version: 20220220.1
2022-02-23T10:20:13.4832606Z Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20220220.1/images/linux/Ubuntu2004-Readme.md
2022-02-23T10:20:13.4833274Z Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20220220.1
2022-02-23T10:20:13.4833922Z ##[endgroup]
2022-02-23T10:20:13.4834266Z ##[group]Virtual Environment Provisioner
2022-02-23T10:20:13.4834631Z 1.0.0.0-main-20220215-1
2022-02-23T10:20:13.4834948Z ##[endgroup]
2022-02-23T10:20:13.4835866Z ##[group]GITHUB_TOKEN Permissions
2022-02-23T10:20:13.4836624Z Actions: write
2022-02-23T10:20:13.4837127Z Checks: write
2022-02-23T10:20:13.4837464Z Contents: write
2022-02-23T10:20:13.4837792Z Deployments: write
2022-02-23T10:20:13.4838177Z Discussions: write
2022-02-23T10:20:13.4838445Z Issues: write
2022-02-23T10:20:13.4838745Z Metadata: read
2022-02-23T10:20:13.4839064Z Packages: write
2022-02-23T10:20:13.4839324Z Pages: write
2022-02-23T10:20:13.4839671Z PullRequests: write
2022-02-23T10:20:13.4839971Z RepositoryProjects: write
2022-02-23T10:20:13.4840325Z SecurityEvents: write
2022-02-23T10:20:13.4840658Z Statuses: write
2022-02-23T10:20:13.4840924Z ##[endgroup]
2022-02-23T10:20:13.4844759Z Secret source: Actions
2022-02-23T10:20:13.4845275Z Prepare workflow directory
2022-02-23T10:20:13.5755613Z Prepare all required actions
2022-02-23T10:20:13.5951976Z Getting action download info
2022-02-23T10:20:13.9590411Z Download action repository 'actions/checkout@v2' (SHA:ec3a7ce113134d7a93b817d10a8272cb61118579)
2022-02-23T10:20:14.8176525Z Download action repository 'GoogleCloudPlatform/release-please-action@v3' (SHA:f0145e31b50c56905571d69a63bbe997c0ce6dd1)
2022-02-23T10:20:15.6655964Z ##[group]Run actions/checkout@v2
2022-02-23T10:20:15.6656407Z with:
2022-02-23T10:20:15.6656678Z   repository: Xxxxxx/xxxxxx
2022-02-23T10:20:15.6657088Z   token: ***
2022-02-23T10:20:15.6657350Z   ssh-strict: true
2022-02-23T10:20:15.6657630Z   persist-credentials: true
2022-02-23T10:20:15.6657893Z   clean: true
2022-02-23T10:20:15.6658133Z   fetch-depth: 1
2022-02-23T10:20:15.6658338Z   lfs: false
2022-02-23T10:20:15.6658571Z   submodules: false
2022-02-23T10:20:15.6658836Z ##[endgroup]
2022-02-23T10:20:15.9459188Z Syncing repository: Xxxxxx/xxxxxx
2022-02-23T10:20:15.9462427Z ##[group]Getting Git version info
2022-02-23T10:20:15.9479001Z Working directory is '/home/runner/work/xxxxxx/xxxxxx'
2022-02-23T10:20:15.9479822Z [command]/usr/bin/git version
2022-02-23T10:20:15.9574107Z git version 2.35.1
2022-02-23T10:20:15.9599069Z ##[endgroup]
2022-02-23T10:20:15.9607048Z Deleting the contents of '/home/runner/work/xxxxxx/xxxxxx'
2022-02-23T10:20:15.9611355Z ##[group]Initializing the repository
2022-02-23T10:20:15.9614972Z [command]/usr/bin/git init /home/runner/work/xxxxxx/xxxxxx
2022-02-23T10:20:15.9704916Z hint: Using 'master' as the name for the initial branch. This default branch name
2022-02-23T10:20:15.9706311Z hint: is subject to change. To configure the initial branch name to use in all
2022-02-23T10:20:15.9707514Z hint: of your new repositories, which will suppress this warning, call:
2022-02-23T10:20:15.9708085Z hint: 
2022-02-23T10:20:15.9708576Z hint: 	git config --global init.defaultBranch <name>
2022-02-23T10:20:15.9708841Z hint: 
2022-02-23T10:20:15.9709274Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2022-02-23T10:20:15.9709798Z hint: 'development'. The just-created branch can be renamed via this command:
2022-02-23T10:20:15.9710822Z hint: 
2022-02-23T10:20:15.9711262Z hint: 	git branch -m <name>
2022-02-23T10:20:15.9732192Z Initialized empty Git repository in /home/runner/work/xxxxxx/xxxxxx/.git/
2022-02-23T10:20:15.9746082Z [command]/usr/bin/git remote add origin https://github.com/Xxxxxx/xxxxxx
2022-02-23T10:20:15.9795620Z ##[endgroup]
2022-02-23T10:20:15.9796712Z ##[group]Disabling automatic garbage collection
2022-02-23T10:20:15.9800560Z [command]/usr/bin/git config --local gc.auto 0
2022-02-23T10:20:15.9839599Z ##[endgroup]
2022-02-23T10:20:15.9861283Z ##[group]Setting up auth
2022-02-23T10:20:15.9862548Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-02-23T10:20:15.9894142Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-02-23T10:20:16.0276708Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-02-23T10:20:16.0309789Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-02-23T10:20:16.0580570Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2022-02-23T10:20:16.0629502Z ##[endgroup]
2022-02-23T10:20:16.0630757Z ##[group]Fetching the repository
2022-02-23T10:20:16.0639298Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +a2627ce2b29254616b15d9cf595360d4ad38eb29:refs/remotes/origin/main
2022-02-23T10:20:16.6994171Z remote: Enumerating objects: 513, done.        
2022-02-23T10:20:16.6998445Z remote: Counting objects:   0% (1/513)        
[...]     
2022-02-23T10:20:16.7321519Z remote: Counting objects: 100% (513/513)        
2022-02-23T10:20:16.7322290Z remote: Counting objects: 100% (513/513), done.        
2022-02-23T10:20:16.7322601Z remote: Compressing objects:   0% (1/452)        
[...]   
2022-02-23T10:20:16.7693779Z remote: Compressing objects: 100% (452/452), done.        
2022-02-23T10:20:16.7733819Z Receiving objects:   0% (1/513)
[...]
2022-02-23T10:20:17.1909197Z Receiving objects:  99% (508/513)
2022-02-23T10:20:17.1927917Z remote: Total 513 (delta 49), reused 316 (delta 17), pack-reused 0        
2022-02-23T10:20:17.1942704Z Receiving objects: 100% (513/513)
2022-02-23T10:20:17.1943745Z Receiving objects: 100% (513/513), 15.12 MiB | 35.93 MiB/s, done.
2022-02-23T10:20:17.1949396Z Resolving deltas:   0% (0/49)
[...]
2022-02-23T10:20:17.1994046Z Resolving deltas: 100% (49/49), done.
2022-02-23T10:20:17.2390834Z From https://github.com/Xxxxxx/xxxxxx
2022-02-23T10:20:17.2391856Z  * [new ref]         a2627ce2b29254616b15d9cf595360d4ad38eb29 -> origin/main
2022-02-23T10:20:17.2414782Z ##[endgroup]
2022-02-23T10:20:17.2415594Z ##[group]Determining the checkout info
2022-02-23T10:20:17.2417258Z ##[endgroup]
2022-02-23T10:20:17.2417960Z ##[group]Checking out the ref
2022-02-23T10:20:17.2422785Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2022-02-23T10:20:17.3045580Z Switched to a new branch 'main'
2022-02-23T10:20:17.3045991Z branch 'main' set up to track 'origin/main'.
2022-02-23T10:20:17.3057058Z ##[endgroup]
2022-02-23T10:20:17.3103202Z [command]/usr/bin/git log -1 --format='%H'
2022-02-23T10:20:17.3129352Z 'a2627ce2b29254616b15d9cf595360d4ad38eb29'
2022-02-23T10:20:17.3329762Z ##[group]Run GoogleCloudPlatform/release-please-action@v3
2022-02-23T10:20:17.3330058Z with:
2022-02-23T10:20:17.3330470Z   token: ***
2022-02-23T10:20:17.3330671Z   command: manifest
2022-02-23T10:20:17.3330870Z   default-branch: main
2022-02-23T10:20:17.3331046Z   fork: false
2022-02-23T10:20:17.3331223Z   clean: true
2022-02-23T10:20:17.3331428Z   bump-minor-pre-major: false
2022-02-23T10:20:17.3331829Z   bump-patch-for-minor-pre-major: false
2022-02-23T10:20:17.3332070Z   monorepo-tags: false
2022-02-23T10:20:17.3332245Z   draft: false
2022-02-23T10:20:17.3332445Z   draft-pull-request: false
2022-02-23T10:20:17.3332647Z ##[endgroup]
2022-02-23T10:20:17.8664451Z ✔ Building pull requests
2022-02-23T10:20:17.8667074Z ✔ Building strategies by path
2022-02-23T10:20:18.3924067Z ##[error]release-please failed: Not Found
2022-02-23T10:20:18.4065752Z Post job cleanup.
2022-02-23T10:20:18.5269956Z [command]/usr/bin/git version
2022-02-23T10:20:18.5321440Z git version 2.35.1
2022-02-23T10:20:18.5363754Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-02-23T10:20:18.5399725Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-02-23T10:20:18.5628469Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-02-23T10:20:18.5658886Z http.https://github.com/.extraheader
2022-02-23T10:20:18.5660108Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2022-02-23T10:20:18.5688072Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-02-23T10:20:18.6143745Z Cleaning up orphan processes
```

### Additional information

`.release-please-manifest.json`

```json
{
  "assets/": "0.3.1",
  ".": "0.3.1"
}
```

`release-please-config.json`

```json
{
  "packages": {
    "assets/": {
      "releaseType": "node",
      "packageName": "xxxxxx-fe",
      "draft": false,
      "bumpMinorPreMajor": true,
      "bumpPatchForMinorPreMajor": true,
      "changelogPath": "assets/CHANGELOG.md",
      "versioning": "default"
    },
    ".": {
      "releaseType": "elixir",
      "packageName": "xxxxxx",
      "draft": false,
      "bumpMinorPreMajor": true,
      "bumpPatchForMinorPreMajor": true,
      "changelogPath": "CHANGELOG.md",
      "versioning": "default"
    }
  }
}
```

**Comment by wearethefoos** [comment]
I dug around a bit more and found that the problem is that you can't mix packages and a root project (`.`) in a manifest. Not sure if that's something you want to support, so feel free to close this.

**Comment by bcoe** [comment]
> I dug around a bit more and found that the problem is that you can't mix packages and a root project (.) in a manifest. Not sure if that's something you want to support, so feel free to close this.

This should be supported, I do so myself [here](https://github.com/googleapis/google-api-nodejs-client).

Bother you to open a ticket on the main [release-please](https://github.com/googleapis/release-please), if possible with a minimal reproduction.

**Comment by xiaoyu-tamu** [comment]
This error also occur when I add new entry to `release-please-config.json`
